aksel.nav.no/website/components/token-docs/token/example/FontToken.tsx

```diff
@@ -0,0 +1,47 @@
+import { LineHeightIcon } from "@navikt/aksel-icons";
+import { Heading, VStack } from "@navikt/ds-react";
+import { TokenForDocumentationT } from "../../../types/tokens";
+
+const FontToken = ({ token }: { token: TokenForDocumentationT }) => {
+  let fontStyling = {
+    fontSize: "var(--ax-font-size-heading-medium)",
+    fontFamily: "var(--ax-font-family)",
+    fontWeight: "var(--ax-font-weight-regular)",
+    lineHeight: "var(--ax-font-line-height-heading-xsmall)",
+  };
+  switch (token.group) {
+    case "family":
+      fontStyling = {
+        ...fontStyling,
+        fontFamily: token.value,
+      };
+      break;
+    case "size":
+      fontStyling = {
+        ...fontStyling,
+        fontSize: token.value,
+      };
+      break;
+    case "weight":
+      fontStyling = {
+        ...fontStyling,
+        fontWeight: token.value,
+      };
+      break;
+    case "line-height":
+      return (
+        <VStack as="div" align="center" justify="center" height="100%">
+          <LineHeightIcon width="32px" height="32px" title={token.name} />
+        </VStack>
+      );
+  }
+  return (
```

Comment:
Burde enten ligge som default case i switchen, eller utelates/håndteres som i ColorToken.tsx?

Reply:
Denne returnen brukes i alle tilfeller, men la til en console.warn for default case over, og setter vk-error klassen om default case brukes også.